tests/integration/targets/info_test/run.yml

```diff
@@ -1,19 +1,34 @@
-- hosts: localhost
-  gather_facts: no
+- name: Run Info Test Tasks
+  hosts: localhost
+  gather_facts: false
   collections:
     - community.general
-  vars_files:
-    - vars.yml
 
   tasks:
-    - name: Vcsim
-      ansible.builtin.import_role:
-        name: prepare_soap
 
-    - name: Vcsim
+    - name: Import eco-vcenter credentials
+      ansible.builtin.include_vars:
+        file: ../../../../scripts/integration_config.yml
```

Comment:
Please move this var file to locate the integration dir

Reply:
fixed, I moved them to be under integration folder and without the scripts folder